website/business.html

```diff
@@ -0,0 +1,169 @@
+
+        <!-- Favicon Data -->
+        <link rel="apple-touch-icon" sizes="57x57" href="assets/icons/apple-icon-57x57.png">
+        <link rel="apple-touch-icon" sizes="60x60" href="assets/icons/apple-icon-60x60.png">
+        <link rel="apple-touch-icon" sizes="72x72" href="assets/icons/apple-icon-72x72.png">
+        <link rel="apple-touch-icon" sizes="76x76" href="assets/icons/apple-icon-76x76.png">
+        <link rel="apple-touch-icon" sizes="114x114" href="assets/icons/apple-icon-114x114.png">
+        <link rel="apple-touch-icon" sizes="120x120" href="assets/icons/apple-icon-120x120.png">
+        <link rel="apple-touch-icon" sizes="144x144" href="assets/icons/apple-icon-144x144.png">
+        <link rel="apple-touch-icon" sizes="152x152" href="assets/icons/apple-icon-152x152.png">
+        <link rel="apple-touch-icon" sizes="180x180" href="assets/icons/apple-icon-180x180.png">
+        <link rel="icon" type="image/png" sizes="192x192" href="assets/icons/android-icon-192x192.png">
+        <link rel="icon" type="image/png" sizes="32x32" href="assets/icons/favicon-32x32.png">
+        <link rel="icon" type="image/png" sizes="96x96" href="assets/icons/favicon-96x96.png">
+        <link rel="icon" type="image/png" sizes="16x16" href="assets/icons/favicon-16x16.png">
+        <link rel="manifest" href="assets/icons/manifest.json">
+        <meta name="msapplication-TileColor" content="#ffffff">
+        <meta name="msapplication-TileImage" content="assets/icons/ms-icon-144x144.png">
+        <meta name="theme-color" content="#ffffff">
+        <!-- End Favicon Data -->
+</head>
+<body>
+    <div class="section-separator">
+        <img class="section-separator-background" src="./assets/images/border6-night.svg">
+      </div>
+      
+      <section id="sponsors">
+        <div class="container">
+          <div class="row">
+            <div id="business-header" class="section-header col-xs-12">
+              <div class="header-text">Business Track Registration</div>
+              <div class="header-bar"></div>
+            </div>
+          </div>
+          <div class="row">
+                <ul class="stepList offset-md-1">
+                  <li>
+                    <div class="step">Step 1</div> 
+                    <span><strong>Go through our standard registration process.</strong>
+                      <div class="register-btn">
+                          <a href="https://app.hackpsu.org" target="_blank">
+                            <div class="header-text text-center">
+                              <p class="text-center">Register Here</p>
+                            </div>
+                          </a>
+                        </div>
+                    </span>
+                  </li>
+                  <li>
+                    <div class="step">Step 2</div> 
+                    <span><strong>Go to your User Profile by clicking the icon in the top-right corner of the screen after registration.</strong></span></li>
```

Comment:
```suggestion
                    <span><strong>Click <a href="https://app.hackpsu.org/profile>here</a> to go to sign-up for the business challenge after registration.</strong></span></li>
```
Or something like that